python/cudf/cudf/core/column/column.py

```diff
@@ -1186,6 +1186,12 @@ def as_column(arbitrary, nan_as_null=None, dtype=None, length=None):
             data = data.astype(dtype)
     # TODO: Remove nvstrings here when nvstrings is fully removed
     elif isinstance(arbitrary, nvstrings.nvstrings):
+        warnings.warn(
+            "nvstrings will be removed in 0.15. \
+            Please use equivalent from libcudf",
+            DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
Could move this message to a variable and use it in multiple places, but doesn't really matter since we're getting rid of it next release anyway.

Reply:
as you mentioned we will be getting rid of this in next release, so don't think beautifying would be of any help.